src/profile-logic/marker-data.js

```diff
@@ -854,3 +854,70 @@ export function removeNetworkMarkerURLs(payload: NetworkPayload) {
   payload.URI = '';
   payload.RedirectURI = '';
 }
+
+export function getMarkerFullDescription(marker: Marker) {
+  let name = marker.name;
```

Comment:
We can use `description` as the new variable name to reflect the function name change.